environment.yml

```diff
@@ -0,0 +1,99 @@
+name: sba
```

Comment:
Change this back to slimmed down pip reqs. It works better for people using different venvs.